src/client/config.js

```diff
@@ -0,0 +1,30 @@
+export class USER_URL {
```

Comment:
Great idea to keep a file for global strings and other values.

Rather than export classes with static fields, simply use one plain old JavaScript object containing properties and values.

This avoids the unnecessary complication of classes and all that comes with them.